tests/spec/fs.unlink.spec.js

```diff
@@ -104,3 +104,22 @@ describe('fs.unlink', function() {
     });
   });
 });
+
+/**
+ * fsPromises tests
+ */
+describe('fsPromises.stat', function() {
+  beforeEach(util.setup);
+  afterEach(util.cleanup);
+  it('should return an error if trying to delete a file that does not exist', function(done) {
+    var fsPromises = util.fs().promises;
+
+    fsPromises.unlink('/myFile')
```

Comment:
To make this more promise-friendly, please return this function.

Reply:
thank you for your suggestion :)